src/content/docs/en/guides/content-collections.mdx

```diff
@@ -536,205 +536,11 @@ const { Content } = await entry.render();
-        const filteredPosts = allPosts.filter((post) =>
-          post.data.tags.includes(tag)
-        );
-        return {
-          params: { tag },
-          props: { posts: filteredPosts },
-        };
-      });
-    }
-    
-    const { tag } = Astro.params;
-    const { posts } = Astro.props;
-    ---
-
-    <BaseLayout pageTitle={tag}>
-        <p>Posts tagged with {tag}</p>
-        <ul>
-            { posts.map((post) => <BlogPost url={'/posts/' + post.slug} title={post.data.title} />) }
-        </ul>
-    </BaseLayout>
-    ```
-
-    The same logic appears in the tag index page, which becomes:
-    
-    ```astro title="src/pages/tags/index.astro" "post.data" "getCollection(\"posts\")"
-    ---
-    import { getCollection } from "astro:content";
-    import BaseLayout from "../../layouts/BaseLayout.astro";
-    const allPosts = await getCollection("posts");
-    const tags = [...new Set(allPosts.map((post) => post.data.tags).flat())];
-    const pageTitle = "Tag Index";
-    ---
-    ...
-    ```
-
-    :::note
-    Any individual Markdown or MDX file imports should be replaced by [`getEntry()`](/en/reference/api-reference/#getentry).
-    :::
-    
-8. Update the code that uses the publish date in the `layouts/MarkdownPostLayout.astro` file.
-
-    Previously, `pubDate` was a string. Now, after introducing types for your posts' frontmatter, `pubDate` is a `Date`.
-    To render the date, convert it to a string:
-
-    ```astro title="src/layouts/MarkdownPostLayout.astro" "frontmatter.pubDate.toDateString()"
-    ...
-    <BaseLayout pageTitle={frontmatter.title}>
-      <p>{frontmatter.pubDate.toDateString()}</p>
-      <p><em>{frontmatter.description}</em></p>
-      <p>Written by: {frontmatter.author}</p>
-      <img src={frontmatter.image.url} width="300" alt={frontmatter.image.alt} />
-    ...
-    ```
-    
-
-    Lastly, the tutorial blog project includes an RSS feed. This function must also use `getCollection` and the `data` object, and be converted to an async function to do so:
-
-    ```js title="src/pages/rss.xml.js" {4-5, 10-15} 
-    import rss from "@astrojs/rss";
-    import { getCollection } from "astro:content";
-
-    export async function GET() {
-      const posts = await getCollection('posts');
-      return rss({
-        title: 'Astro Learner | Blog',
-        description: 'My journey learning Astro',
-        site: 'https://my-blog-site.netlify.app',
-        items: posts.map((post) => ({
-          title: post.data.title,
-          pubDate: post.data.pubDate,
-          description: post.data.description,
-          link: `/posts/${post.slug}/`,
-        })),
-        customData: `<language>en-us</language>`,
-      });
-    }
-    ```
-
-For the full example of the blog tutorial using content collections, see the [Content Collections branch](https://github.com/withastro/blog-tutorial-demo/tree/content-collections) of the tutorial repo.
+## Migrating from File-Based Routing
```

Comment:
Noting that [the final page of the tutorial](https://docs.astro.build/en/tutorial/6-islands/3/#next-steps) links here. May be better to update that page to link to the new page?